src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 0.1.16 (2017-09-11)
 +++++++++++++++++++
 * webapp: able to create a webapp in a resource group other than the service plan's
+* webapp: able to update and show authentication settings using "az webapp auth update/show"
```

Comment:
This should go in a section `unreleased` not under a version that has already been released.